webpack/assets/javascripts/react_app/components/UserJwt/components/UserJwtForm.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { Grid } from 'patternfly-react';
+import PropTypes from 'prop-types';
+
+import NumericInput from '../../common/forms/NumericInput';
+import Select from '../../common/forms/Select';
+import { translate as __ } from '../../../common/I18n';
+
+const UserJwtForm = ({ handleSubmit }) => {
+  const expirationUnits = [
+    { label: __('Never'), value: 'never' },
+    { label: __('Hour'), value: 'hours' },
+    { label: __('Day'), value: 'days' },
+    { label: __('Week'), value: 'weeks' },
+    { label: __('Month'), value: 'months' },
+  ];
+
+  const [expirationUnit, setExpirationUnit] = useState('never');
+  const [expirationValue, setExpirationValue] = useState('');
+
+  const changeUnit = u => {
+    if (u === 'never') {
+      setExpirationValue('');
+    }
+    setExpirationUnit(u);
+  };
+
+  return (
+    <div>
+      <Grid.Row>
+        <Grid.Col md={8}>
+          <Select
+            label={__('Expire in')}
+            className="jwt-expiration-value"
+            options={expirationUnits}
+            value={expirationUnit}
+            onChange={e => changeUnit(e.target.value)}
+          />
+        </Grid.Col>
+      </Grid.Row>
+      <Grid.Row>
+        <Grid.Col md={8}>
+          <NumericInput
+            label={__('Value')}
+            className="jwt-expiration-value"
+            minValue={1}
+            onChange={value => setExpirationValue(value)}
```

Comment:
I think it can go shorter, the value will be passed automatically :)
```suggestion
            onChange={setExpirationValue}
```